content/tutorials/drift-detection-and-remediation/index.md

```diff
@@ -0,0 +1,224 @@
+---
+title_tag: "Detect and Remediate Infrastructure Drift"
```

Comment:
Why don't we push through changes to the deployments tutorials title as part of this PR. Yeah I like what you did to the title on this one. Its much more use cases focused, thanks for doing it

Reply:
going to add this as part of a new PR,, this one's now way behind main